bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SvgImageCreatorImp.java

```diff
@@ -297,6 +297,8 @@ private boolean generateSvgImage(File imagePresentationDir, UploadedPresentation
             if(tempPng.exists()) {
                 tempPng.delete();
             }
+
+            done = true;

```

Comment:
It seems it will reach this point even when failing. What do you think about verifying if the svg was generated?

```suggestion
            if(destsvg.exists()) {
                done = true;
            }
```